shared/teams/team/settings-tab/retention/index.js

```diff
@@ -200,6 +201,17 @@ class _RetentionPicker extends React.Component<Kb.PropsWithOverlay<Props>, State
             Individual channels can override this.
           </Kb.Text>
         )}
+        {this.props.policyIsExploding && (
```

Comment:
This is above 'Individual channels can override this.' in designs. Also the gap between them is too big in app
<img width="310" alt="image" src="https://user-images.githubusercontent.com/11968340/52222015-8c7d9780-2870-11e9-93fa-851b1f9c1de3.png">
design:
<img width="319" alt="image" src="https://user-images.githubusercontent.com/11968340/52222032-9606ff80-2870-11e9-8ebe-e3bdc19c3ee5.png">